### PR TITLE
feat(overmind-graphql): support file uploads

### LINKED ISSUE #446: Overmind-graphql support file upload
`graphql-request` added support for file uploads in version `3.1.0`. It would be good to upgrade that package in `overmind-graphql` as well in order to support file uploads.

### PR DESCRIPTION
**Pull request by mi-kas**
Closes #446

### CODE REVIEW

**Comment by mi-kas** [comment]
@christianalfoni Wdyt?

**Comment by christianalfoni** [comment]
Hi @mi-kas and so sorry for my late reply! I am preparing a release now. Just one question. Why does it need `cross-fetch`? 😄

**Comment by mi-kas** [comment]
There wass a bug in the `graphql-request` release `3.1.0` (https://github.com/prisma-labs/graphql-request/issues/206) and adding `cross-fetch` solved the issue. Let me double check if the issue is fixed with `graphql-request@3.2.0`

**Comment by eXtreme** [comment]
@mi-kas @christianalfoni Friendly inquiry, is there any progress on this? Anything I can do to help? I could really use file upload feature in one of my projects.

**Comment by mi-kas** [comment]
@eXtreme @christianalfoni I bumped `graphql-request` and was able to remove thereby the `cross-fetch` dependency. Let me know if this works for you

**Comment by eXtreme** [comment]
@mi-kas Upload works fine for me 👍 

From travis and my local build, it looks like `package-lock.json` needs to be updated with `graphql-request@3.3.0`.

**Comment by mi-kas** [comment]
@eXtreme Thanks for the feedback! Lock file is updated now.

@christianalfoni PR is ready for review

File: package.json [hunk 1]
```diff
@@ -61,7 +61,8 @@
     "tslib": "1.10.0",
     "vscode": "^1.1.36",
     "vue": "3.0.0-rc.10",
-    "ws": "7.0.0"
+    "ws": "7.0.0",
+    "cross-fetch": "3.0.6"
```
Comment (SergeiKalachev) [inline package.json:65]: @mi-kas Hello my dear friend!
Do we still need this line? I've seen you managed to get rid of `cross-fetch` dependency by bumping `graphql-request` version.

Comment (mi-kas) [reply package.json]: @SergeiKalachev Good catch my friend 👍  Removed now this dependency

**Comment by mi-kas** [comment]
@christianalfoni Any feedback on this PR?

**Review by christianalfoni — APPROVED** [review]
Looks good to me, lets try this 😄

**Pull request merged**